masterplan_tools/specification/specification_validator.py

```diff
@@ -0,0 +1,27 @@
+"""
+The module is aimed to validate input json data according to various schemas.
+"""
+from pathlib import Path
+import json
+import os
+from jsonschema import validate, RefResolver, Draft4Validator
+
+package_directory = os.path.dirname(os.path.abspath(__file__))
+
+
+class SpecificationValidator:
+    """
+    todo
+    """
+
+    def __init__(self):
+        schemas = (json.load(open(source))
+                   for source in Path(os.path.join(package_directory, "schemas")).iterdir())
```

Comment:
Those lines are not formatted properly
And you should never use `open(source)` without setting it to a variable and closing after. Of cource, garbage collector will close file for you most of the times, but it is just a bad practice of working with such things without context managers:
```python
with open(source) as f:
    json.load(f)
```
However, the structure above does not fit in a loop. So i recommend using:
`schemas = [json.loads(source.read_text() for source in (Path(package_directory) / "schemas").glob("*.json")]` instead.